forge/src/lib.rs

```diff
@@ -22,3 +22,29 @@ mod test_helpers;
 
 /// The Forge EVM backend
 pub use foundry_evm::*;
+
+/// Metadata on how to run fuzz/invariant tests
+#[derive(Debug, Clone, Copy, Default)]
+pub struct TestOptions {
+    /// Whether fuzz tests should be run
+    pub include_fuzz_tests: bool,
+    /// The number of test cases that must execute for each fuzz test
+    pub fuzz_runs: u32,
+    /// The maximum number of global test case rejections allowed
+    /// by proptest, to be encountered during usage of `vm.assume`
+    /// cheatcode.
+    pub fuzz_max_local_rejects: u32,
+    /// The maximum number of local test case rejections allowed
+    /// by proptest, to be encountered during usage of `vm.assume`
+    /// cheatcode.
+    pub fuzz_max_global_rejects: u32,
+    /// The number of runs that must execute for each invariant test group.
+    pub invariant_runs: u32,
+    /// The number of calls executed to attempt to break invariants in one run.
+    pub invariant_depth: u32,
+    /// Fails the invariant fuzzing if a revert occurs
+    pub invariant_fail_on_revert: bool,
+    /// Allows overriding an unsafe external call when running invariant tests. eg. reetrancy
+    /// checks
+    pub invariant_call_override: bool,
```

Comment:
should we expose these under foundry toml in `invariants`, similar to how the `fmt` section looks like?

Reply:
Not yet IMO. I still want to spend some time thinking on the best way to structure the toml so we can support https://github.com/foundry-rs/foundry/issues/744 without cheatcodes, which will apply to fuzz params and invariant params